canonical-web-and-design/vanilla-framework
# Add cookie policy to Vanilla site

## Done

Add cookie policy
*Note: running locally, the site only loads one page and then the connection starts dropping?*

Fixes https://github.com/canonical-web-and-design/web-squad/issues/3265

## QA

- Pull code
- Run `./run`
- Open http://0.0.0.0:8101/
- See the new cookie popup and try the settings
- Test the "Manage your tracker settings" link from the footer
